standalone/README.md

```diff
@@ -35,41 +35,53 @@ If this image is out-of-date, you can edit it [here](https://docs.google.com/dra
 
 ## Setting Up `simple-server`
 
-Below are instructions to setting up simple-server on a set of servers. These instructions assume that you have already
-provisioned servers and have their static IP addresses available. If you don't have servers provisioned yet, you will
-need to do so first.
+These instructions assume that you have already provisioned servers and have their static IP addresses available.
+If you don't have servers provisioned yet, you will need to do so first. See [provisioning testing servers.](#provisioning-testing-servers)
```

Comment:
Nitpick of the week: Can we move the `.` outside the hyperlink? :smile:

Reply:
Believe it or not, I did think about this for a whole five seconds and decided it was best for the period to be inside.

I was wrong and of course [Yes. There Are Punctuation Rules for Hyperlinks](http://writingspaces.org/wwsg/punctuation-hyperlinks).